apps/dashboard/src/app/services/help-boxes.service.ts

```diff
@@ -98,12 +98,12 @@ const TOUR_STEPS: IStepOption[] = [
     content: 'tour.step6.content',
     stepId: '84px,-64px,right-top',
   },
-  // {
-  //   anchorId: 'step7',
-  //   title: 'tour.step7.title',
-  //   content: 'tour.step7.content',
-  //   stepId: '84px,-43px,right-top',
-  // },
+  {
+    anchorId: 'step7',
+    title: 'tour.step7.title',
+    content: 'tour.step7.content',
+    stepId: '84px,-43px,right-top',
```

Comment:
did you check if this position was still valid now there is a new menu for it?

Reply:
I'm not sure.
I just check it, it's correct.